Makefile

```diff
@@ -7,13 +7,11 @@ BUILD_DATE             = $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
 GIT_COMMIT             = $(shell git rev-parse HEAD)
 GIT_REMOTE             = origin
 GIT_BRANCH             = $(shell git rev-parse --symbolic-full-name --verify --quiet --abbrev-ref HEAD)
-GIT_TAG                = $(shell git describe --exact-match --tags HEAD 2>/dev/null || git rev-parse --short=8 HEAD 2>/dev/null)
+GIT_TAG                = $(shell git describe --tags --abbrev=0)
 GIT_TREE_STATE         = $(shell if [ -z "`git status --porcelain`" ]; then echo "clean" ; else echo "dirty"; fi)
 
 export DOCKER_BUILDKIT = 1
 
-# To allow you to build with or without cache for debugging purposes.
-DOCKER_BUILD_OPTS     := --no-cache
```

Comment:
no longer needed, all builds are done an Github Actions - we should always cache